Added new listener to the status of the message request.

teamagam3 opened the pull request.

teamagam3 committed:
Added snackbar on new message response (success and failure)

review-ninja commented:
<a href="https://app.review.ninja/teamagam/GimelGimel/pull/94" target="_blank"><img src="https://app.review.ninja/52010471/pull/94/badge" alt="ReviewNinja"/>

teamagam3 committed:
CR Fixes

Removed static methods from the GGMessageSender, and added "global" instance to GGApplcation (Use getGGMessageSender() method to retrieve it).
